.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+on: push
+
+name: Continuous Integration
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
```

Comment:
p5.

Should we update this?

Reply:
Node v18 is in lts mode, so the version update is not necessary for now.